tools/includes/proceed_p.sh

```diff
@@ -14,16 +16,22 @@ fi
 # Returns success if "yes", failure if "no" or non-interactive.
 function proceed_p {
 	if ! $INTERACTIVE; then
-		echo "$1 Aborting" >&2
+		error "$1 Aborting"
 		return 42
 	fi
 	local OK
 
 	# Clear input before prompting
 	while read -r -t 0 OK; do read -r OK; done
 
-	[[ "$1" ]] && M="$1 " || M=""
-	read -n 1 -p "$M${2:-Proceed?} [y/N] " OK
+	local PROMPT
+	[[ -n "$1" ]] && PROMPT="$1 "
+	PROMPT="${PROMPT}${2:-Proceed?} [y/N] "
+	if color_supported; then
+		PROMPT=$(FORCE_COLOR=1 info "$PROMPT")
```

Comment:
```suggestion
		PROMPT=$(FORCE_COLOR=1 prompt "$PROMPT")
```

Set a different color to stand out for a question.

Reply:
Sure, no objection.